pkg/action/mount.go

```diff
@@ -434,12 +442,7 @@ func SelinuxRelabel(cfg *types.RunConfig, spec *types.MountSpec) error {
 		return nil
 	}
 
-	extraMounts := map[string]string{
-		"/run":  "/run",
-		"/proc": "/proc",
-	}
-
-	return utils.ChrootedCallback(&cfg.Config, spec.Sysroot, extraMounts, func() error {
+	return utils.ChrootedCallback(&cfg.Config, spec.Sysroot, nil, func() error {
```

Comment:
There was no need to bind mount `/proc` as this is already bind mounted as part of any chroot (we bind mount, at least, `/sys`, `/dev` and `/proc` for any chrooted call). There is no need to relabel `/run` as this happens anyway when SELinux policy is loaded.

Reply:
Yep, `/run` was mounted because in an earlier iteration the relabel file was written in the chroot, and after moving the writing out I forgot to also remove the extra mounts, good catch! :+1: